pkg/ccl/changefeedccl/cdceval/expr_eval_test.go

```diff
@@ -290,57 +330,84 @@ CREATE TABLE foo (
 			actions: []string{
 				"INSERT INTO foo (a, b) VALUES (1, '   spaced out      ')",
 			},
-			predicate: "SELECT btrim(b), parse_timetz('1:00-0') AS past FROM _",
+			stmt: "SELECT btrim(b), parse_timetz('1:00-0') AS past FROM _",
 			expectMainFamily: []decodeExpectation{
 				{
 					keyValues: []string{"   spaced out      ", "1"},
 					allValues: map[string]string{"btrim": "spaced out", "past": "01:00:00+00:00:00"},
 				},
 			},
 		},
+		// TODO(yevgeniy): enable this test.
+		// This requires adding support to "fetch" those magic system columns from
+		// row fetcher, or replace them with a function call.
+		//{
+		//	testName:   "main/magic_column",
+		//	familyName: "main",
+		//	actions: []string{
+		//		"INSERT INTO foo (a, b) VALUES (1,  'hello')",
+		//	},
+		//	stmt: "SELECT a,  crdb_internal_mvcc_timestamp FROM foo",
+		//	expectMainFamily: []decodeExpectation{
+		//		{
+		//			keyValues: []string{"hello", "1"},
+		//			allValues: map[string]string{"a": "1", "crdb_internal_mvcc_timestamp": "xxx"},
+		//		},
+		//	},
+		//},
 		{
-			testName:   "main/btrim_wrong_type",
+			testName:   "main/trigram",
 			familyName: "main",
 			actions: []string{
-				"INSERT INTO foo (a, b) VALUES (1, '   spaced out      ')",
+				"INSERT INTO foo (a, b) VALUES (1,  'hello')",
 			},
-			predicate: "SELECT btrim(a) FROM _",
+			stmt: "SELECT a,  b % 'hel' AS trigram FROM foo",
```

Comment:
The current bug with string % is that it always returns true when the expression is being evaluated in a job, because it depends on a session variable that defaults to 0.3 but isn't being set so it's using a similarity threshold of 0. I think this PR probably doesn't fix this, so I think either remove this test or (if I'm wrong) add a test case where it's false using an enterprise feed like environment.

Reply:
Ack; going to disable the test.